### ISSUE TITLE
Content Generator about Callisto Network

### ISSUE THREAD

**Issue by KotNaplakal** [EthereumCommonwealth/Proposals#25]
# Proposal

An application to attract new members to the community and generate content about Callisto Network.

# Description

My team can create and deploy a smart contract (including a web application to interact with it) that will help anyone who wants to receive reimbursement for the organization of the event dedicated to Callisto Network.

Main functions and conditions:

The event organizer writes information about the planned event to the smart contract by filling out a special form in the web application.
Based on the information entered from the total budget of the smart contract, the required amount is reserved for reimbursement of the costs of organizing the event.
The number of participants at the event must be at least 10.
The amount of reimbursement depends on the number of participants + the number of photos / videos taken and is set by the founders of Callisto Network.
The founders of the network determine the dependence of the maximum number of reports and the number of declared participants.
Reimbursement of expenses for organizing events according to the approved format will be made only after the provision of photo / video reports.

Such phrases as, Callisto Network, Blockchain technology, Cryptocurrency, Smart-Contracts, Decentralized File Storage and cold staking protocol, must be present on posters, flyers and other promotional materials, as well as the logo of our project.
This data can be changed / updated.
All advertising materials must be well-designed in accordance with the requirements imposed on this kind of products.

All participants of the event should say a few words about the project under the video recording and be photographed against the background of the posters.
At the event there should be at least speeches by speakers on 3 approved topics with a duration of at least 30 minutes each.
Presentations should consist of a general report and answers to questions.
Speeches should be recorded on video and posted on Youtube.
The photos are published in a web application.
Links to photo / video materials are reports for moderators.
The number of links to photo / video materials that the event organizer can send is limited depending on the number of participants.
The amount of reimbursement for the organization of the event is determined by the number of photo / video materials accepted by the moderator.

Moderators will be offered and approved by community members who also hold at least 10,000 CLO and have accounts on the BTT forum.
The number of votes is determined by the status of the account:
Junior Member - 1 vote.
Member - 2 votes.
Full member - 3 votes.
Senior member - 4 votes.
Legendary - 5 votes.

The moderator receives a reward of at least 1 CLO per view of 1 link to the photo / video content that is paid from the total budget of the smart contract to the address specified when registering the moderator.
The amount of remuneration to the moderators is established by the founders of the network.
Moderators can be re-elected in the event of a prolonged inactivity or the adoption of incorrect reports on the event.
The moderator should review and evaluate the quality of photo / video reports, their uniqueness and hide photos from the web application that do not meet the accepted criteria.
The owner of a smart contract serves as a super moderator and ensures that moderators do not accept reports already paid to other event organizers.
A supermoderator with a reason can initiate a vote for the removal of authority from the moderator.
The moderator gets the right to moderating the event randomly.
The number of moderators is limited and is established by voting.

Moderators receive a reward after approval by the supermoderator.
The founders of the network supplement the balance of the smart contract, thereby determining the size of the budget.
The owner of a smart contract can not manage the budget.
The founder of the network can withdraw CLO from the balance of the smart contract.
CLO reserved for events can not be withdrawn.

P.S. It is advisable to hold all meetups simultaneously, on the appointed date. In this case, there will be a qualitative jump in community activity in social networks and on exchange, which can lead to a significant increase in the cost of CLO.

**Distributed in this way CLO should be frozen in a smart contract for at least 6 months.**

# Funding goal

1500000 CLO. My address: 0x5505B49bDCe1513B08Cb0d4Cfd2566E4c14579FC
Payment plan.
0 CLO after the approval of the detailed technical assignment.
300,000 CLO after deployment a smart contract.
200000 CLO after the start of the web application work.
1000000 CLO after moderation of 100,000 links to photo / video content.

**Comment by sneg55** [comment]
We not looking for event marketing at the moment.

**Comment by AnonymizerEthereum** [comment]
I do not understand ... You are now paying 20,000 for 4 articles or 1 video, but do not want to get at least 14 videos and 10 photos for the same amount?

**Comment by mCLOster** [comment]
This means that I can come to the club and hold a themed party with reimbursement of expenses for the organization? The more participants of the party, the more CLO I get?

**Comment by AnonymizerEthereum** [comment]
Yes, we thought about this option.
But we also thought about private parties at the organizer's home, in a friendly atmosphere.
Imagine: Photos and videos from around the world, tens of thousands of links to the project ...

**Comment by mCLOster** [comment]
Q: Do you still need help except programmers and auditors? If so, how can the community still support you except to promote Callisto?

We always need help, promoting CLO in your social networks, writing blog posts, creating meetups about Callisto, etc. You could open a proposal to help Callisto in https://github.com/EthereumCommonwealth/Proposals/issues

http://telegra.ph/Callisto-Discord-AMA-07092018-07-09

**Comment by AnonymizerEthereum** [comment]
@mCLOster I did it. Our project will help to organize meetups en masse.

**Comment by KotNaplakal** [comment]
> Давайте расширяться, когда, если не сейчас?

В нашем полку прибыло!) Рад, что прибавляется людей понимающих, что нужно увеличивать сообщество и объем исходящего от него контента!

**Comment by Dexaran** [comment]
I think that this proposal is no longer relevant.

**Comment by KotNaplakal** [comment]
> Я думаю, что это предложение уже не актуально.
@Dexaran 
Можно кое что изменить и будет снова актуально. Кратко если, то можно на этих митапах продавать CLO новым участникам, средства от продажи направлять в фонд и из него компенсировать расходы организаторов. Главное это то, что проданные CLO надо заморозить в смарт контракте например на 6 месяцев, а за это держателям начислить хорошие проценты. В таком случае, сообщество будет увеличено, организаторы заработают, курс возрастет.
